elixirkit/demo/mix.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Demo.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :demo,
+      version: "0.1.0",
+      elixir: "~> 1.14",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Demo.Application, []}
+    ]
+  end
+
+  defp deps do
+    [
+      {:elixirkit, path: ".."}
+    ]
+  end
```

Comment:
Add a TODO that, once you release the package, you need to include the `elixirkit_swift` and similar directories to the package. Unless you think they would be separate packages in their respective package mangers?

Reply:
Definitely not separate packages. Good call.